src/core/server/saved_objects/import/import_saved_objects.test.ts

```diff
@@ -362,6 +362,17 @@ describe('importSavedObjects()', () => {
         },
       })),
     });
+
+    const clientResponse = {
+      saved_objects: [
+        { id: '1', type: 'index-pattern', attributes: { title: 'My Index Pattern' } },
```

Comment:
Can we also change these titles to verify the existing title matching in the test.